src/fsharp/xlf/FSStrings.cs.xlf

```diff
@@ -1422,6 +1422,11 @@
         <target state="translated"> Nespárované prvky se budou ignorovat.</target>
         <note />
       </trans-unit>
+      <trans-unit id="EnumMatchIncomplete1">
+        <source>Enums may take values outside known cases.</source>
+        <target state="translated">Enums may take values outside known cases.</target>
```

Comment:
These shouldn't be in the 'translated' state - same for all of the non-English translations.

Reply:
Should it be "new"?